src/scipp/typing.py

```diff
@@ -12,7 +12,7 @@ def is_scalar(obj: _std_typing.Any) -> bool:
     """
     Return True if the input is a scalar.
     """
-    return obj.dims == []
+    return obj.dims == ()
```

Comment:
The fact that we had to change this indicates that we should probably have used `ndim() == 0` for this check.

Reply:
Yes, that makes sense.